frontend/packages/console-shared/src/components/spotlight/Spotlight.scss

```diff
@@ -0,0 +1,64 @@
+@keyframes expand {
+  0% {
+    outline-offset: -4px;
+    outline-width: 4px;
+    opacity: 1;
+  }
+  100% {
+    outline-offset: 21px;
+    outline-width: 12px;
+    opacity: 0;
+  }
+}
+
+@keyframes fade-in {
```

Comment:
```suggestion
@keyframes co-spotlight-fade-in {
```

Reply:
done

Reply:
@christianvogt @nemesis09 Shouldn't this be `ocs-spotlight-expand` since this is in `console-shared` package?

Reply:
Not sure it matters... we have a poor consistency across the console for keyframes and their names having the appropriate module. I think it's because they are only ever used in the one file. I'm indifferent either way though.

Reply:
yes it should be. my mistake

Reply:
enamed to ocs-spotlight-expand